app/controllers/organizations/forms_controller.rb

```diff
@@ -0,0 +1,13 @@
+class Organizations::FormsController < Organizations::BaseController
+  before_action :verified_staff
+  
+  layout "dashboard"
+    
+  def edit
+    @form = Current.organization.form || Form.create(
```

Comment:
Conditionally creating the form seems suboptimal. Are we better off with a seed or something that we _know_ will run? We only want one form per organization.

Reply:
Yeh if we are able to add a callback to auto create the Form on organization create would be good imo.